cmd/kubelet/app/server.go

```diff
@@ -1397,6 +1397,8 @@ func getCgroupDriverFromCRI(ctx context.Context, s *options.KubeletServer, kubeD
 				continue
 			}
 			// CRI implementation doesn't support RuntimeConfig, fallback
+			legacyregistry.MustRegister(kubeletmetrics.CRILosingSupport)
```

Comment:
Maybe move the registration out of the retry loop?

Reply:
at this point it's effectively out of the loop. Once we decide the error is unimplemented we fail, and that's the only place we could register this metric unless we saved that failure in a variable and checked it later. I prefer this